docs/requirements-wasm.txt

```diff
@@ -1,2 +1,2 @@
-pyodide-build==0.23.3
-pydantic<2
+# This determines the pyodide version
+pyodide-build==0.24.1
```

Comment:
I can be convinced that an exact version of Pyodide is needed, since extensions are vendored into it.